### PR TITLE
Adjustments for new UVO page (#8).

### PR DESCRIPTION
**Pull request by martinsabo**
Prvý nástrel.

### CODE REVIEW

File: lib/uvobot/uvo_scraper.rb [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 module Uvobot
   class UvoScraper
-    SEARCH_URL = 'https://www2.uvo.gov.sk/evestnik?p_p_id=evestnik_WAR_eVestnikPortlets&p_p_lifecycle=1' \
-                 '&p_p_state=normal&p_p_mode=view&p_p_col_id=column-2&p_p_col_pos=1&p_p_col_count=2'.freeze
-    NEW_ISSUE_URL = 'https://www2.uvo.gov.sk/evestnik/-/vestnik/aktual'.freeze
-    IT_CONTRACTS_CODE = 72
+    UVO_ROOT_URL = 'https://www.uvo.gov.sk'.freeze
+    SEARCH_URL = "#{UVO_ROOT_URL}/vestnik/oznamenia/zoznam".freeze
+    NEW_ISSUE_URL = "#{UVO_ROOT_URL}/vestnik-a-registre/vestnik-479.html".freeze
```
Comment (jsuchal) [inline lib/uvobot/uvo_scraper.rb]: `BULLETIN_URL`

File: spec/uvobot/uvo_parser_spec.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 require './lib/uvobot/uvo_parser'
+require './lib/uvobot/uvo_scraper'
```
Comment (jsuchal) [inline spec/uvobot/uvo_parser_spec.rb]: Kvoli tomu testu tu nemusis tahat zavislost. Staci tam lubovolna ina konstanta.

File: lib/uvobot/uvo_scraper.rb [hunk 3]
```diff
@@ -3,10 +3,10 @@
...
+    IT_CONTRACTS_CODE = '48000000-8 72000000-5'.freeze
```
Comment (jsuchal) [inline lib/uvobot/uvo_scraper.rb]: Dobry postreh!

File: lib/uvobot/uvo_parser.rb [hunk 4]
```diff
@@ -2,28 +2,34 @@
...
+    def self.parse_detail_link(tr_node, bulletin_url)
+      bulletin_url + tr_node.attributes['onclick'].text.scan(/'(.*)'/).first[0]
+    end
+
     def self.parse_detail(html)
       # there are multiple formats of detail page, this method does not handle them all for now
       detail = {}
       h_doc = doc(html)
-      amount_node = h_doc.xpath('//div[text()="Hodnota "]').css('span').first
+      amount_node = h_doc.xpath('//div[text()="Hodnota            "]').css('span').first
```
Comment (jsuchal) [inline lib/uvobot/uvo_parser.rb]: Tu by som z hodnoty tahal vsetky tie spany. Lebo mena moze byt ina, cena z/bez dph.

File: lib/uvobot/notifications/discourse_notifier.rb [hunk 5]
```diff
@@ -46,7 +46,7 @@ def detail_message(detail)
         if detail
           "**Cena:** #{detail[:amount]} EUR  "
         else
-          "**Detaily sa nepodarilo extrahovať.**  "
+          '**Detaily sa nepodarilo extrahovať.**  '
```
Comment (jsuchal) [inline lib/uvobot/notifications/discourse_notifier.rb]: Este do toho `.join` o poschodie vyssie pridaj aj tie 2 medzery a nemusi to byt tu.

Comment (jsuchal) [inline lib/uvobot/notifications/discourse_notifier.rb]: Preco su tam vlastne tie medzery?

Comment (martinsabo) [inline lib/uvobot/notifications/discourse_notifier.rb]: Markdown linebreak. "When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return."

File: lib/uvobot/uvo_parser.rb (hunk 4 above)
Comment (martinsabo) [inline lib/uvobot/uvo_parser.rb]: Mozem (zajtra).

File: lib/uvobot/notifications/discourse_notifier.rb (hunk 5 above)
Comment (jsuchal) [inline lib/uvobot/notifications/discourse_notifier.rb]: aha chapem.

**Comment by jsuchal** [comment]
:+1:

**Pull request merged**